index.html

```diff
@@ -12,6 +12,24 @@
 </head>
 
 <body>
+    <div class="top-main">
+        <header>                
+            <nav>
+                <a href="index.html">
```

Comment:
try putting to the a tag with the logo in it in a div right outside of the nav tag and then flexbox on the header tag